substrate/frame/im-online/src/benchmarking.rs

```diff
@@ -64,34 +62,55 @@ pub fn create_heartbeat<T: Config>(
 	Ok((input_heartbeat, signature))
 }
 
-benchmarks! {
-	#[extra]
-	heartbeat {
-		let k in 1 .. MAX_KEYS;
-		let (input_heartbeat, signature) = create_heartbeat::<T>(k)?;
-	}: _(RawOrigin::None, input_heartbeat, signature)
-
-	#[extra]
-	validate_unsigned {
-		let k in 1 .. MAX_KEYS;
-		let (input_heartbeat, signature) = create_heartbeat::<T>(k)?;
+#[benchmarks]
+mod benchmarks {
+	use super::*;
+
+	#[benchmark(extra)]
+	fn heartbeat(k: Linear<1, MAX_KEYS>) -> Result<(), BenchmarkError> {
+		let (input_heartbeat, signature) = create_heartbeat::<T>(k.into())?;
```

Comment:
```suggestion
		let (input_heartbeat, signature) = create_heartbeat::<T>(k)?;
```
Are you sure that `into()` is actually needed? It shouldn't be, the same applies below